packages/element/src/create-interpolate-element.js

```diff
@@ -137,7 +118,9 @@ const createInterpolateElement = ( interpolatedString, conversionMap ) => {
 	tokenizer.lastIndex = 0;
 
 	if ( ! isValidConversionMap( conversionMap ) ) {
-		return interpolatedString;
+		throw new TypeError(
+			`The conversionMap provided is not valid.  It must be an object with values that are WPElements`
```

Comment:
there's an extra space between the sentences that doesn't belong.

Reply:
A sure indicator of the age of someone writing that 😆 (still haven't shaken fully muscle memory from the days of typewriters) - see cb0e8fb